app/screens/select_server/select_server.js

```diff
@@ -137,8 +137,8 @@ class SelectServer extends PureComponent {
         });
     }
 
-    handleLoginOptions = (props) => {
-        const {config, intl, license, theme} = props;
+    handleLoginOptions = () => {
+        const {config, intl, license, theme} = this.props;
```

Comment:
doesn't this receive the props cause in `componentWillUpdate` we pass **nextProps** ?

Reply:
@jarredwitt ^